tests/core/full_node/test_full_node.py

```diff
@@ -845,15 +851,46 @@ async def test_new_transaction_and_mempool(self, wallet_nodes):
 
         await time_out_assert(10, new_transaction_not_requested, True, incoming_queue, new_transaction)
 
+        # Cannot resubmit transaction
+        status, err = await full_node_1.full_node.respond_transaction(
+            successful_bundle, successful_bundle.name(), peer, test=True
+        )
+        assert status == MempoolInclusionStatus.FAILED
+        assert err == Err.ALREADY_INCLUDING_TRANSACTION
```

Comment:
I don't understand this. How is `force_high_fee` related to this response to the peer?
Sending a response to a peer here seems new too.

Reply:
The force_high_fee means that that transaction is guaranteed to go into the block that we make. Th way we know that it goes is, is first we try to submit the transaction again, but it will fail because it's already in the mempool.

Then, after making the block, we submit the transaction again, and we will not get ALREADY_INCLUDING_TRANSACTION, we will get a different error. This is because when updating the mempool for the block, we remove traces of old transactions from the seen map. 

This is necessary, because when we do a reorg (reverting tx block) we want to be able top resubmit, which we also test

Reply:
I see. I hadn't realized this was a test when reviewing this. This makes sense now.